src/containers/AccountConnection.jsx

```diff
@@ -43,6 +43,21 @@ class AccountConnection extends Component {
     this.store.addConnectionStatusListener(konnector, this.connectionListener)
   }
 
+  getFolderPathIfNecessary (connector, account) {
```

Comment:
I think so, it will check if this connector need a folder as well. I wanted to make it explicit.